Dockerfile

```diff
@@ -1,27 +1,40 @@
 FROM opensuse/tumbleweed:latest
-RUN zypper ref && zypper in -y git zip wget docker ruby gzip make jq curl which unzip direnv
+# Catapult dependencies:
+RUN zypper ref && zypper in --no-recommends -y git zip wget docker ruby gzip make jq python-yq curl which unzip bazel1.2 direnv
 RUN echo 'eval $(direnv hook bash)' >> ~/.bashrc
+
+RUN wget "https://github.com/krishicks/yaml-patch/releases/download/v0.0.10/yaml_patch_linux" -O yaml-patch
+RUN mv yaml-patch /usr/local/bin && chmod +x /usr/local/bin/yaml-patch
+
 # Extras, mostly for the terminal image (that could be split in another image)
-RUN zypper in -y vim zsh tmux glibc-locale glibc-i18ndata python ruby python3 python3-pip
+RUN zypper in --no-recommends -y vim zsh tmux glibc-locale glibc-i18ndata python ruby python3 python3-pip cf-cli
+
+RUN zypper ar --priority 100 https://download.opensuse.org/repositories/devel:/languages:/go/openSUSE_Factory/devel:languages:go.repo && \
+  zypper --gpg-auto-import-keys -n in --no-recommends -y --from=devel_languages_go go1.13
+
+RUN zypper ar --priority 100 https://download.opensuse.org/repositories/Cloud:Tools/openSUSE_Tumbleweed/Cloud:Tools.repo && \
+  zypper --gpg-auto-import-keys -n in --no-recommends -y kubernetes-client
 
+# k8s backends dependencies:
+RUN zypper in --no-recommends -y terraform aws-cli aws-iam-authenticator
 
-RUN zypper ar https://download.opensuse.org/repositories/devel:/languages:/go/openSUSE_Factory/devel:languages:go.repo
-RUN zypper --gpg-auto-import-keys -n in --from=devel_languages_go go1.13
+RUN curl -o google-cloud-sdk.tar.gz https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-264.0.0-linux-x86_64.tar.gz && \
```

Comment:
it will be nice if all our tools go in same place i.e /usr/local/bin
 example: 
```
curl https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz > /tmp/google-cloud-sdk.tar.gz \
    && mkdir -p /usr/local/bin/gcloud \
    && tar -C /usr/local/bin/gcloud -xvf /tmp/google-cloud-sdk.tar.gz \
    && /usr/local/bin/gcloud/google-cloud-sdk/install.sh --quiet
```

Reply:
The case of `gcloud` is convoluted, as it needs all the google-cloud-sdk folder. And putting that full folder on /usr/local/bin may make a lot of things appear in the path when they are not supposed to be there.

For the case of  `az`. it's not better either; they do the same thing, create a python virtualenv, and afterwards add a specific folder to the path via bash.

I'm gonna leave it for now, it could be cleaned, yes. But recreating the Dockerfile locally takes ages..